chemicaltoolbox/openduck/openduck_chunk.xml

```diff
@@ -0,0 +1,79 @@
+<tool id="openduck_chunk" name="OpenDUck chunk" version="@VERSION@">
+    <description>for dynamic undocking</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        cp '$ligand' ./ligand.sdf &&
+        duck_chunk
+            -p '$protein'
+            -l ./ligand.sdf
+            -c '$cutoff'
+            $ignore_buffers
+            ## -i is interaction, e.g. A_ASP_156_OD2
+            -i '${ia.chain_sel}'_'${ia.res_sel}'_'${ia.resid_sel}'_'${ia.atom_sel}' &&
+        tar -czf allfiles.tar.gz --exclude=allfiles.tar.gz *
```

Comment:
this taring can be conditioned with the `return_tar`

Reply:
Whoops, will add